docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md

```diff
@@ -488,18 +488,18 @@ In this section, we will:
   $(element).on('dropon', function(ev, drop, drag) {...});
   ```
 
-  Notice that `drag` is the 3nd argument to the event.  You can listen to
-  `drop` events in [can-stache] and pass the `drag` argument to a function like:
+  Notice that `drag` is the 2nd argument to the event.  You can listen to
+  `drop` events in [can-stache], and pass the `drag` argument to a function, like:
 
   ```html
   ($dropon)="addVideo(%arguments[2])"
   ```
 
 - You will need to associate the drag objects with the video being
-  dragged so when a `drop` happens you can know the video being dropped. The
+  dragged so, when a `drop` happens, you can know which video is being dropped. The
```

Comment:
This is correct, and we could probably make it better by saying “…dragged so you can know which video is being dropped when a `drop` happens.”